dbms/src/IO/CodecDeflateQpl.h

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+#ifdef ENABLE_QPL_COMPRESSION
```

Comment:
sorry, where have defined the macro `ENABLE_QPL_COMPRESSION` ?

Reply:
See my comment: https://github.com/pingcap/tiflash/pull/7364#issuecomment-1605860762